gridstatus/caiso.py

```diff
@@ -268,10 +304,62 @@ def _process_solar_and_wind_forecast_dam(self, data):
                     "Wind": "Wind MW",
                 },
             ),
-            ["Interval Start", "Interval End", "Location"],
+            ["Interval Start", "Interval End", "Publish Time", "Location"],
+        )
+
+        df.columns.name = None
+
+        return df.sort_values(
+            ["Interval Start", "Publish Time", "Location"],
+        ).reset_index(drop=True)
+
+    def _add_forecast_publish_time(
```

Comment:
in the future, we might need to rename this function if we support other markets besides dam that have different logic